### PR TITLE
dl/driving license

### PR DESCRIPTION
**Pull request by darrikonn**
Use a formal naming approach for easier comparison between applications:

### tree on src
image

### CODE REVIEW

File: libs/application/templates/driving-lessons/src/forms/DrivingLessonsApplication.ts [hunk 1]
```diff
@@ -226,7 +226,7 @@ export const DrivingLessonsApplication: Form = buildForm({
             buildSubmitField({
               id: 'submit',
               placement: 'footer',
-              name: 'sick',
```
Comment (arnifreyrs) [inline libs/application/templates/driving-lessons/src/forms/DrivingLessonsApplication.ts:229]: nooooooo :D

**Comment by darrikonn** [comment]
This shouldn't be reviewed by the Aranja team.

I believe your codeownership is overwriting ours:
```
/libs/application/                                        @island-is/aranja
```

^ This is too broad, might want to narrow it down.

File: libs/application/templates/driving-license/src/forms/application.ts [hunk 2]
```diff
@@ -132,6 +132,35 @@ export const drivingLicenseApplication: Form = buildForm({
...
+      children: [
+        buildMultiField({
+          id: 'healthDeclaration',
+          name: 'Heilbrigðisyfirlýsing',
+          children: [
+            buildIntroductionField({
+              id: 'intro',
+              name: '',
+              introduction:
+                'Ef sótt er um réttindi í flokkum <b>AM</b>, <b>A1</b>, <b>A2</b>, <b>A</b>, <b>B</b>, <b>BE</b> eða <b>T</b> nægir heilbrigðisyfirlýsing ein og sér, nema sýslumaður telji þörf á læknisvottorði eða ef umsækjandi hefur náð 65 ára aldri eða hann vilji heldur skila læknisvottorði. Með umsókn um aðra flokka ökuréttinda (aukin ökuréttindi) er krafist læknisvottorðs á sérstöku eyðublaði.',
```
Comment (arnifreyrs) [inline libs/application/templates/driving-license/src/forms/application.ts:147]: you can also use markdown if you want (see `introIntroduction` in `reference-template/src/forms/messages.ts` if you want an example)

**Comment by arnifreyrs** [comment]
@darrikonn yes I agree, will change it in a PR later today.

File: libs/application/templates/driving-license/src/lib/template.ts [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+const dataSchema = z.object({
+  address: z.object({
+    home: z.string().nonempty(),
+    postcode: z.string(),
+    city: z.string().nonempty(),
+  }),
+  user: z.object({
+    name: z.string().nonempty(),
+    phoneNumber: z.string().min(7),
+    nationalId: z.string().refine((x) => kennitala.isPerson(x)),
```
Comment (arnifreyrs) [inline libs/application/templates/driving-license/src/lib/template.ts:26]: this is great!

**Review by arnifreyrs — APPROVED** [review]
Sick

File: libs/application/templates/driving-license/src/forms/review.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import {
+  buildForm,
+  buildIntroductionField,
+  buildMultiField,
+  buildSection,
```
Comment (dabbeg) [inline libs/application/templates/driving-license/src/forms/review.ts:5]: All of the forms are importing a lot of stuff they are not using

File: libs/application/templates/driving-license/src/forms/application.ts (hunk 2 above)
Comment (darrikonn) [reply libs/application/templates/driving-license/src/forms/application.ts:147]: aaah nice!!!

**Review by dabbeg — APPROVED** [review]
I like it!

File: libs/application/templates/driving-license/src/forms/review.ts (hunk 4 above)
Comment (darrikonn) [reply libs/application/templates/driving-license/src/forms/review.ts:5]: true... will fix it in the next slice, since the design isn't ready.

**Pull request merged**